GoldenGMSA/Program.cs

```diff
@@ -116,13 +134,30 @@ static void ProcessComputePwdOptions(ComputePwdOptions options)
                 if (options.Sid == null)
                     throw new ArgumentNullException(nameof(options.Sid));
 

```

Comment:
This cancels the offline mode and tries to talk to AD even when all the required arguments have been passed on the command line (sid/KdsRootKeyBase64/ManagedPwdIdBase64). Offline mode does not require the domain/forest names.